SU2_CFD/include/numerics/turbulent/turb_sources.hpp

```diff
@@ -35,18 +35,19 @@
  */
 struct CSAVariables {
   /*--- List of constants ---*/
-  const su2double cv1_3 = pow(7.1, 3);
-  const su2double k2 = pow(0.41, 2);
-  const su2double cb1 = 0.1355;
-  const su2double cw2 = 0.3;
-  const su2double ct3 = 1.2;
-  const su2double ct4 = 0.5;
-  const su2double cw3_6 = pow(2, 6);
-  const su2double sigma = 2.0 / 3.0;
-  const su2double cb2 = 0.622;
-  const su2double cb2_sigma = cb2 / sigma;
-  const su2double cw1 = cb1 / k2 + (1 + cb2) / sigma;
-  const su2double cr1 = 0.5;
+  const su2double
+   cv1_3 = pow(7.1, 3.0),
+   k2    = pow(0.41, 2.0),
+   cb1   = 0.1355,
+   cw2   = 0.3,
```

Comment:
revert please, it is clang-format or nothing

Reply:
That was the reason to have a `const double` in front of all of those @suargi, in order to have clang-format not collapse these together